WcaOnRails/app/views/registrations/_entry_fee.html.erb

```diff
@@ -1,5 +1,21 @@
+<%
+  label ||= t('registrations.payment_form.labels.entry_fees')
+  money_amount ||= @registration.entry_fee
+  hint ||= ""
+  hint_context ||= ""
+  hint_context_attribute = hint_context.blank? ? "" : "class=\"text-#{hint_context} bg-#{hint_context}\""
+%>
+
 <% if @registration.entry_fee %>
-  <%= alert :info do %>
-    <%= t 'registrations.entry_fee', fee: entry_fee %>
-  <% end %>
+  <div class="simple_form form-horizontal">
+    <div class="form-group">
+      <label class="col-sm-2 control-label" for="entry_fees"><%= label %></label>
+      <div class="col-sm-9">
+        <p class="form-control-static"><%= humanized_money_with_symbol(money_amount) %></p>
+      <% unless hint.blank? %>
```

Comment:
Indentation seems a bit off here. The `unless hint.blank?` is inside of the `div.col.sm-9`, right?